regifter.sql

```diff
@@ -15,104 +15,114 @@ CREATE DATABASE regifter;
 -- giver - string
 -- value - integer
 -- previously_regifted boolean
-
+CREATE TABLE gifts (id SERIAL PRIMARY KEY, gift TEXT NOT NULL, giver TEXT, value INT, previously_regifted BOOLEAN);
 
 -- 
 \echo See details of the table you created
 -- 
-
+\dt 
 
 -- 
 \echo Alter the table so that the column price is changed to value 
 -- 
-
-
+--starting name is already value, removed and added to get practice in
+ALTER TABLE gifts DROP COLUMN value;
+--\d gifts;
+ALTER TABLE gifts ADD COLUMN value INT;
+--\d gifts
 -- 
 \echo Insert a peach candle, given by 'Santa' thats value is 9 and has been previously regifted
 -- 
-
-
+INSERT INTO gifts (gift, giver, previously_regifted, value) VALUES ('peach candle', 'Santa', TRUE, 9);
+--SELECT * FROM gifts;
 --
 \echo Query for all the columns in your gifts table
 -- 
-
+SELECT * FROM gifts;
 
 --
 \echo Uncomment below to insert 5 more gifts
 -- 
 
--- INSERT INTO gifts (gift, giver, value, previously_regifted)
--- VALUES
--- ('peach candle', 'Santa', '9', TRUE),
--- ('cinnamon candle', 'Nick', '19', TRUE),
--- ('soap on a rope', 'Rudolf', '29', FALSE),
--- ('potpurri', 'Elf on the Shelf', '39', TRUE),
--- ('mango candle', 'The Boss', '49', FALSE)
--- ;
+INSERT INTO gifts (gift, giver, value, previously_regifted)
+VALUES
+('peach candle', 'Santa', '9', TRUE),
+('cinnamon candle', 'Nick', '19', TRUE),
+('soap on a rope', 'Rudolf', '29', FALSE),
+('potpurri', 'Elf on the Shelf', '39', TRUE),
+('mango candle', 'The Boss', '49', FALSE)
+;
 
 -- 
 \echo Insert 5 more gifts of your own choosing,  include 1 more candle
 --
-
+INSERT INTO gifts (gift, giver, value, previously_regifted)
+VALUES
+('peach candle', 'Santa', '9', TRUE),
+('tie', 'John', '20', TRUE),
+('mouse', 'Aisha', '10', FALSE),
+('keyboard', 'An Elf', '38', TRUE),
+('guitar', 'Santa', '250', TRUE)
+;
 
 
 --
 \echo Query for gifts with a price greater than or equal to 20
 --
-
+SELECT * FROM gifts WHERE value >= 20;
 
 --
 \echo Query for every gift that has the word candle in it, only show the gift column
 --
-
+SELECT gift FROM gifts WHERE gift LIKE '%candle%';
```

Comment:
Be aware that this will not return "Brown Candle" due to the capital C.

The ILIKE constraint will give you a CASE-INSENSITIVE comparison should you need it